OpenOrchestrator/orchestrator/constants_tab.py

```diff
@@ -146,7 +151,7 @@ def double_click_credential_table(event: tkinter.Event, cred_table: ttk.Treeview
     """
     row = cred_table.identify_row(event.y)
     if row:
-        name, value, _ = cred_table.item(row, 'values')
+        name, value, _, _ = cred_table.item(row, 'values')
```

Comment:
```suggestion
        name, value, *_ = const_table.item(row, 'values')
```

(cf. https://peps.python.org/pep-3132/)